### PR TITLE
ensure css and js assets are present and can be generated  on composer based installs

### PR DESCRIPTION
**Pull request by mollux**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://github.com/mautic/recommended-project/pull/18#issuecomment-1127838121 

#### Description:

When using the composer-recommended project, the `mautic-form.js` is empty, which is not correct.

```
bash-5.1# ls -al docroot/media/js/
total 2392
drwxr-xr-x    2 mautic     mautic          4096 May 16 21:28 .
drwxr-xr-x    7 mautic     mautic          4096 May 16 21:28 ..
-rw-r--r--    1 mautic     mautic        393625 May 16 21:28 app.js
-rw-r--r--    1 mautic     mautic       2041536 May 16 21:28 libraries.js
-rw-r--r--    1 mautic     mautic             0 May 16 21:28 mautic-form.js
```

This is because reads the `mautic:assets:generate` command reads the source file [from the media folder](https://github.com/mautic/mautic/blob/49f042d9cb5aa623310b81772f43a04db2460616/app/bundles/CoreBundle/Command/GenerateProductionAssetsCommand.php#L46).

But that file isn't scaffolded in composer, resulting in an empty file.

This issue was raised in https://github.com/mautic/recommended-project/pull/18#issuecomment-1127838121

next to that, there are css files missing too

```
bash-5.1# ls -al docroot/media/css
total 692
drwxr-xr-x    2 dlp      dlp           4096 May 17 09:46 .
drwxr-xr-x    7 dlp      dlp           4096 May 17 09:46 ..
-rw-r--r--    1 dlp      dlp         134578 May 17 09:46 app.css
-rw-r--r--    1 dlp      dlp         563514 May 17 09:46 libraries.css
```

This PR ensures that all the media (source) files are present.

(A better approach in the future is #10682, as that tackles this in a more generic way)

#### Steps to test this PR:

testing this without #11039 is not simple, as the console (to test the asset generation)  is broken.  
So I suggest we wait for that PR to land, 

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11164"></a>

### CODE REVIEW

**Comment by RCheesley** [comment]
Now the PR #11039 is merged, do we just apply this PR and run `m:a:g` or is there some way we should test?

**Comment by mollux** [comment]
@RCheesley 

Yeah, kind of same issue as other PR: it adds scaffolded files, so you can't really test it. Best approach is to execute the steps in the tests.yml file (see the changed file in this PR) and verify the same files are present in the tarball media folder and composer media folder

**Comment by escopecz** [comment]
@mollux where are the changes in `app/assets/js/mautic-form-src.js` coming from? I don't see how are they related to the composer issue.

**Comment by mollux** [comment]
@escopecz this is the version of those files synced from the media folder to the scaffold folder. There is currently no automated way that keeps those files in sync.

https://github.com/mautic/mautic/pull/10682 will fix this in the future

**Review by escopecz — DISMISSED** [review]
Thanks! 👍  I see no issues in the code

File: app/assets/js/mautic-form-src.js [hunk 1]
```diff
@@ -56,7 +56,7 @@
                 replaceArgs.data['style'] = typeof(replaceArgs.data['style']) == 'undefined' ? 'embed' : replaceArgs.data['style'] ;
                 if (Core.debug()) console.log(replaceArgs.data['style']+' Mautic Form: '+replaceText);
 
-                //display form according with style
+                //display form accoding with style
```
Comment (RCheesley) [inline app/assets/js/mautic-form-src.js]:
```suggestion
                //display form according with style
```

**Review by RCheesley — DISMISSED** [review]
Small typo in the comment which I have committed, otherwise OK. 

@mollux will the GitHub Action which checks the scaffolded files pick this up in the future?

**Comment by mollux** [comment]
@RCheesley fixing that typo in 1 place will trigger a test failure :) it will need fixing in the media folder too

**Comment by RCheesley** [comment]
> @RCheesley fixing that typo in 1 place will trigger a test failure :) it will need fixing in the media folder too

Argh yes, sorry I just realised that 🤦‍♀️

**Review by escopecz — APPROVED** [review]

**Pull request merged**